src/router.js

```diff
@@ -47,6 +79,22 @@ const ROUTES = [
       import('./views/data/data-view.js');
       BNRouter.selectTab(TABS.SANSKRIT);
       document.querySelector('vaadin-drawer-toggle').removeAttribute('style');
+      document
+        .querySelector('img.logo-buddhanexus')
+        .setAttribute('src', '/src/assets/img/buddhanexus_skt.jpg');
+      document
```

Comment:
this can be a reusable function like `showLogo()`. Right now code is unnecessarily 
repeated in this file.